tests/python/unittest/test_runtime_module_export.py

```diff
@@ -203,7 +204,10 @@ def verify_multi_c_mod_export():
         synthetic_cpu_lib.import_module(f)
         synthetic_cpu_lib.import_module(engine_module)
         kwargs = {"options": ["-O2", "-std=c++17", "-I" + header_file_dir_path.relpath("")]}
-        synthetic_cpu_lib.export_library(path_lib, fcompile=False, **kwargs)
+        work_dir = temp.relpath("work_dir")
+        os.mkdir(work_dir)
+        synthetic_cpu_lib.export_library(path_lib, fcompile=False, workspace_dir=work_dir, **kwargs)
+        assert os.path.exists(work_dir + "/devc.o")
```

Comment:
use 

`os.path.join(work_dir, "devc.o")` so it can be portable across OS with different separators